src/commands/kit/bundle.command.ts

```diff
@@ -292,6 +295,23 @@ summary: |
   Deno.writeTextFileSync(fileToUpdate, `${metadataHeader}\n${existingText}`);
 }
 
+async function applyKitDocumentationTf(
+  kitPath: string,
+  content: string | undefined,
+) {
+  const documentationTfFile = path.join(kitPath, "documentation.tf");
+  try {
+    await Deno.stat(documentationTfFile);
+  } catch (e) {
+    if (e instanceof Deno.errors.NotFound) {
+      Deno.writeTextFile(
```

Comment:
f: `writeTextFile` returns a Promise, so we should probably `await` (alternatively, use the sync variant: `writeTextFileSync`).

Reply:
Fixed.